app/src/main/java/com/example/mpkAndroid/domain/model/VehicleStaticFactory.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.mpkAndroid.domain.model
+
+import com.example.mpkAndroid.data.model.VehicleEntity
+
+class VehicleStaticFactory {
+    companion object {
+        fun create(vehicleEntity: VehicleEntity): Vehicle {
```

Comment:
Can't we simplify it to this?
```suggestion
        fun create(vehicleEntity: VehicleEntity): Vehicle {
        return Vehicle(
                    vehicleEntity.name,
                    VehicleType.valueOf(vehicleEntity.type.uppercase()),
                    vehicleEntity.x,
                    vehicleEntity.y
            )
        }
```